test/data/noConflict.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+	<script>var QUnit = parent.QUnit</script>
+	<script src="testinit.js"></script>
+	<script src="../../src/sizzle.js"></script>
+	<script>
+		var Sizzle1 = Sizzle;
+	</script>
+	<script src="../../src/sizzle.js"></script>
+	<script>
+		var Sizzle2 = Sizzle;
+	</script>
+</head>
+<body>
+	<script>
+		var doc = parent.document,
+			unframed = [ doc.getElementById( "qunit-fixture" ), doc.body, doc.documentElement ],
+			framed = Sizzle( "*" );		
```

Comment:
We don't need `unframed` at all, and `Sizzle( "*" )` will yield too many elements... maybe just `Sizzle( "head, body" )`.